hack/dind-cluster.sh

```diff
@@ -243,17 +252,28 @@ function enable-local-registry() {
     ${DOCKER_CMD} push ${ip}:${REGISTRY_PORT}/${image}:${tag}
   done
 
-  local image_prefix=$(echo ${images[0]} | cut -d'-' -f1)
-  echo "OPENSHIFT_IMAGE_FORMAT=${ip}:${REGISTRY_PORT}/${image_prefix}-\\\${component}:\\\${version}" >> "${config_root}/dind-env"
+  local error=0
+  $( grep -q "OPENSHIFT_IMAGE_FORMAT" "${config_root}/dind-env" ) || error=1
+  if [[ "${error}" -eq "1" ]]; then
+    local image_prefix=$( ${BUILD_LOCAL_IMAGES_CMD} get-list | grep -v ERROR | head -n 1 | xargs | cut -d '-' -f1 )
+    echo "OPENSHIFT_IMAGE_FORMAT=${ip}:${REGISTRY_PORT}/${image_prefix}-\\\${component}:\\\${version}" >> "${config_root}/dind-env"
+  fi
 }
 
 function get-local-images() {
   local user_requested_images=$1
+  if [[ -z "${user_requested_images}" || "${user_requested_images}" == "none" ]]; then
+      echo ""
+      return 0
+  fi
+
   local local_image_list=$(${BUILD_LOCAL_IMAGES_CMD} get-list | grep -v ERROR | xargs)
   local supported_images=(${local_image_list})
   local -a images=()
 
-  if [[ -n "${user_requested_images}" ]]; then
+  if [[ "${user_requested_images}" == "all" ]]; then
+    images=${supported_images[@]}
```

Comment:
This looks like it was probably supposed to be `images=(${supported_images[@]})` to assign an array rather than a string.  It does not really matter since this code path is fine with either a string or an array, but the assignment definitely *looks* wrong.